File: RSP/1450_Number_of_Students_Doing_Homework_at_a_Given_Time.py
from ast import List


def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
    '''

    go through the list
    check of the startTime is less and equal and endTime is greater and equal 
    that the query time

    if true 
    icrease counter

    '''

    n = len(startTime)
    count = 0
    for i in range(n):
        if startTime[i] <= queryTime and endTime[i] >= queryTime:
            count+=1

    return count

File: RSP/1450_Number_of_Students_Doing_Homework_at_a_Given_Time.cpp
#include <iostream>
#include <vector>

int busyStudent(vector<int>& startTime, vector<int>& endTime, int queryTime) {
        /* 
        go thorugh the two lists
        if start time [i] < query timr and end time[i] > query time
        count++
        
        O(n) - time
        O(1) - space
        */

        int n = startTime.size();
        int count = 0;
        for (int i = 0; i < n ; i++){
            if (startTime[i] <= queryTime && endTime[i] >= queryTime){
                count++;
            }
        }
        return count;

        
    }